### ISSUE TITLE
UI for showing warnings to user

### ISSUE THREAD

**Issue by stevage** [TerriaJS/terriajs#1737]
We have various warnings that it would be nice to show to the user, without a full modal, and more obvious than a `console.log()`.

For instance:
- service unavailable (before it's used), such as proxy, conversion, URL shortening...
- layers blacklisted from CKAN
- 3D unavailable

@RacingTadpole suggests https://react-bootstrap.github.io/components.html#alerts-closeable

**Comment by kring** [comment]
We need to be clear about who these warnings are for.  IMO only the developer of the site wants to see the `service unavailable` warning.  The CKAN blacklist is useful to whoever runs that CKAN site or the layers it references, and maybe to the developer of the TerriaJS site, but end users will find it pretty annoying.  The 3D unavailable is arguably useful to everyone (that's why we currently show it as a modal).

In short, I don't know how we implement something like this without just annoying everyone.

**Comment by stevage** [comment]
> In short, I don't know how we implement something like this without just annoying everyone.

That's why I tagged it UX :)

Seriously, I'm imagining something like a (!) icon that appears in a corner, and when clicked, reveals a message. A way of saying "Something is wrong" without demanding that you do something about it, but giving you a path to more information if you're looking for it.
